frontend/src/components/DeviceHeaderMenu.tsx

```diff
@@ -50,6 +50,13 @@ export const DeviceHeaderMenu: React.FC<{ device?: IDevice; header?: any }> = ({
               exactMatch
               dense
             />
+            <ListItemLocation
+              title="Settings"
+              icon="cog"
+              iconColor="grayDarker"
+              pathname={`/devices/${device.id}/deviceSettings`}
+              dense
+            />
```

Comment:
```suggestion
```
Let's just add these notification settings to the "Edit" tab instead of making a new one.